tasks.py

```diff
@@ -324,3 +324,10 @@ def _replace(pattern):
     run("git commit -a -m 'Automated update for release v{}'".format(version), echo=True)
     run("git tag v{} -m 'See the release notes for details:\n\nhttps://metallb.universe.tf/release-notes/#version-{}-{}-{}'".format(version, version.major, version.minor, version.patch), echo=True)
     run("git checkout main", echo=True)
+
+
+@task
+def unit(ctx):
```

Comment:
Why not `test`? Seems more intuitive to me (and very common).

Reply:
I have no strong opinion.  I guess I was envisioning a future `e2e` target or something similar for other types of tests.  This is always easy to change, though.  I've changed it to `test`.